packages/editor/src/components/projects/CreateProjectDialog.tsx

```diff
@@ -47,12 +55,26 @@ export const CreateProjectDialog = ({ open, onSuccess, onClose }: Props): any =>
   const [error, setError] = useState('')
   const [projectName, setProjectName] = useState('')
 
+  const project = {
```

Comment:
what's the necessity of a temporary project here? is this a placeholder for the newly created project?

Reply:
Yes